test-infra/src/main/java/com/datastax/oss/driver/api/testinfra/ccm/CcmBridge.java

```diff
@@ -278,17 +332,20 @@ public static Builder builder() {
   public static class Builder {
     private int[] nodes = {1};
     private final Map<String, Object> cassandraConfiguration = new LinkedHashMap<>();
+    private final Map<String, Object> dseConfiguration = new LinkedHashMap<>();
     private final List<String> jvmArgs = new ArrayList<>();
     private String ipPrefix = "127.0.0.";
-    private CassandraVersion cassandraVersion = CcmBridge.DEFAULT_CASSANDRA_VERSION;
-    private String cassandraDirectory = CcmBridge.DEFAULT_CASSANDRA_DIRECTORY;
+    private CassandraVersion version = CcmBridge.DEFAULT_CASSANDRA_VERSION;
+    private String directory = CcmBridge.DEFAULT_CASSANDRA_DIRECTORY;
```

Comment:
:+1: on the latter.  Originally I wanted to make the version/directory/dse enablement API configurable, but I think it would be better to avoid tests that use and rely on specifying versions, and rather just only run them when that version is configured.    Will fix.